transformations/dev-prod-mode/dev-prod-mode.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Development/Production Mode Tutorial
+permalink: /transformations/dev-prod-mode/
+---
+
+* TOC
+{:toc}
+
+The feature Development/Production Mode enables you to develop code above your production data without affecting
+or endangering the data in any way. It works as a version system where you create a new branch, make changes 
+in it, and run your configurations as needed. Output data is stored separately and does not overwrite your
+production data. 
+
+In this tutorial, we will show you how Development/Production Mode works, what the process of developing in 
+a branch looks like, and what you need to do before you can start.
+
+## Private Beta Warning
+Development/Production Mode is currently in private beta. Some features may not work as expected. Please bear 
+with us while we polish all necessities. Any feedback is welcome at ideas@keboola.com.
```

Comment:
```suggestion
with us while we polish all necessities. Any feedback is welcome at [ideas@keboola.com](mailto:ideas@keboola.com).
```

Reply:
A taky email mame? Nema to byt odkaz? https://ideas.keboola.com/

Reply:
jo určitě!

Reply:
to mi doplnil Tomas F., mela jsem to zkontrolovat, promin

Reply:
> Nevěř nikomu, komu je nad třicet.

😆